### DISCUSSION TITLE
Can not push on current user github

### DISCUSSION THREAD

**Discussion by Smachy24** [community/community#81894] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey,
I have two accounts, user A and user B.

I'm currently connected with user A : 
git config user.email = user A email
git config user.name = user A name

I'm trying to push on a repo of user A but there is an error : 
_remote: Permission to <User A>/portfolio.git denied to <User B>.
fatal: unable to access 'https://github.com/<User A>/portfolio.git/': The requested URL returned error: 403_

I added user B on user A's repoand it worked. And it was user A who commited on the repo not user B. 
I removed user B from the repo and there is the same error.

What can I do please ?

**Comment by arham4312** [comment, marked as answer]
Remember, changing user.email and user.name in Git config only affects commit authorship, not GitHub authentication.

**1. Cache Credentials:** Sometimes the issue can be with cached credentials on your machine. You can clear these out and re-enter them when prompted. On most systems, you can use the git credential-cache exit command to clear the cache.

**2. Try Fresh Clone:** If issues persist, clone the repository again with user A's credentials.

**Comment by Smachy24** [comment]
With the command it didn't work (_fatal: credential-cache unavailable; no unix socket support_), but it succeed with removing it manually on windows credential manager.

Thank you !

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Smachy24! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
